Services/includes/TMU.h

```diff
@@ -0,0 +1,111 @@
+/*
+ * TMU.h
+ *
+ * Created: 2/24/2020 11:47:11 AM
+ *  Author: Reem
+ */ 
+#ifndef TMU_H_
+#define TMU_H_
+#include "std_types.h"
+#include "Timer.h"
+
+/****************************************************/
+/*              Defines                             */
+/****************************************************/
+//TMU Channels
+#define		TMU_TIMER_CH0					0
+#define		TMU_TIMER_CH1					1
+#define		TMU_TIMER_CH2					2
+
+//TMU Modes
+#define     ONE_SHOT                        0
+#define     PERIODIC                        1
+
+//TMU Resolution
+#define TMU_RESOLUTION                      250
+
+//TMU Buffer size
+#define TMU_TASK_BUFFER_SIZE                100
+
+//ERROR Value Of the Module 
+#define TMU_ERROR                          -700
+
+       
+
+/************************************************************************/
+/*			  Structures Definitions		                            */
+/************************************************************************/
+typedef struct TMU_Cfg{
+	uint8_t TIM_Ch;
+	uint8_t Resolutions;
+}TMU_Cfg;
+typedef struct TMU_TASK_Cfg{
+	uint16_t Task_id;
+	void (*TMU_Cbk_ptr)(void);
```

Comment:
I need a better way to define a pointer to function to be more genaric in case we gonna change the parameters later